addons/actions.md

```diff
@@ -7,8 +7,89 @@ title: Actions
+
+* `setMasterVolume(float volume)`: Sets the volume of the host machine (volume in range 0-1)
+* `increaseMasterVolume(float percent)`: Increases the volume by the given percent
+* `decreaseMasterVolume(float percent)`: Decreases the volume by the given percent
+* `float getMasterVolume()`: Returns the current volume as a float between 0 and 1
+* `playSound(String filename)`: Plays the given sound file. The file must be an mp3 or wav and located in `${openhab.home}/sounds`
+* `playStream(String url)`: Plays the audio stream at the given url
+* `say(String text)`: Says the given text through Text-to-Speech
+* `say(String text, String voice)`: Says the given text through the given voice (depends on the TTS engine and voices installed)
+* ` say(String text, String voice, String device)`: Says the given text through the given voice to the given output device (Mac OSX only).
+
+To get a list of available voices use `say -v ?` and to get a list of devices uses `say -a ?`.
+
+### Logging Actions
+
+* `logDebug(String loggername, String logText)`: Logs logText on level DEBUG to openhab.log
+* `logInfo(String loggername, String logText)`: Logs logText on level INFO to openhab.log
+* `logWarn(String loggername, String logText)`: Logs logText on level WARN to openhab.log
+* `logError(String loggername, String logText)`: Logs logText on level ERROR to openhab.log
+
+`logText` can be a compete String, constructed through concatination, or through arguments.
+
+**Complete String Example:** `logInfo("Garage", "This is a complete String")`
+**Concatination Example:** `logDebug("Lighting", "This is a string concatination:" + Light.name)`
+**Arguments Example:** `logWarn("Alarm", "The {} window has been open for {} hours!", Window.name, timeOpen)`
+
+One can configure whether specific log entries are logged out and where they get logged to (e.g. to a separate file) by [editing the logger configuration]({{base}}/administration/logging.html).
+
+### HTTP Actions
+
+* `sendHttpGetRequest(String url)`: Sends an GET-HTTP request and returns the result as a String
+* `sendHttpPutRequest(String url)`: Sends a PUT-HTTP request and returns the result as a String
+* `sendHttpPutRequest(String url, Sting contentType, String content)`: Sends a PUT-HTTP request with the given content and returns the result as a String
+* `sendHttpPostRequest(String url)`: Sends a POST-HTTP request and returns the result as a String
+* `sendHttpPostRequest(String url, String contentType, String content)`: Sends a POST-HTTP request with the given content and returns the result as a String
+* `sendHttpDeleteRequest(String url)`: Sends a DELETE-HTTP request and returns the result as a String
+
+### Timers
+
+`createTimer(AbstractInstant instant, Procedure procedure)`: schedules a block of code to execute at a future time
+
+`instant` is usually a `DateTime` calculated using the built in variable `now`.
+`procedure` is the block of code and is defined using lambda notation (i.e. square brackets)
```

Comment:
I think I've done this correctly. I made the updates and pushed the changes. I don't need to do anything else to make is show up here as part of the PR, right? 

All suggestions were implemented.